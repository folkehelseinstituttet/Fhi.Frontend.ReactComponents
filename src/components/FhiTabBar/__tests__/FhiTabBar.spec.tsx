import { MemoryRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import TabBar from '../FhiTabBar';

describe('FhiTabBar', () => {
  test('snapshot test', () => {
    const tabBar = renderer.create(
      <MemoryRouter initialEntries={['/']}>
        <TabBar
          className="test"
          links={[
            { title: 'Side 1', path: '/1' },
            { title: 'Side 2', path: '/2' },
            { title: 'Side 3', path: '/3' },
          ]}
        />
      </MemoryRouter>,
    );
    const tree = tabBar.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
