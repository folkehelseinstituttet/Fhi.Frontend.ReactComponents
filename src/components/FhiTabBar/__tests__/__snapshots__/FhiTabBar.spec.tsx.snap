// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FhiTabBar snapshot test 1`] = `
<div
  className="FhiTabBar__NavContainer-sc-2nj45d-2 dhyNbv test"
>
  <div
    className="container"
  >
    <div
      className="row"
    >
      <div
        className="col-12"
      >
        <ul
          className="FhiTabBar__NavList-sc-2nj45d-1 gQnIJP nav nav-tabs"
          onKeyDown={[Function]}
        >
          <a
            className="FhiTabBar__TabBarLink-sc-2nj45d-0 cVQuXe nav-link"
            data-rr-ui-event-key={null}
            disabled={false}
            href="/1"
            onClick={[Function]}
          >
            <div>
              <span>
                Side 1
              </span>
            </div>
          </a>
          <a
            className="FhiTabBar__TabBarLink-sc-2nj45d-0 cVQuXe nav-link"
            data-rr-ui-event-key={null}
            disabled={false}
            href="/2"
            onClick={[Function]}
          >
            <div>
              <span>
                Side 2
              </span>
            </div>
          </a>
          <a
            className="FhiTabBar__TabBarLink-sc-2nj45d-0 cVQuXe nav-link"
            data-rr-ui-event-key={null}
            disabled={false}
            href="/3"
            onClick={[Function]}
          >
            <div>
              <span>
                Side 3
              </span>
            </div>
          </a>
        </ul>
      </div>
    </div>
  </div>
</div>
`;
